site/guide/customize-documentation-templates.qmd

```diff
@@ -3,20 +3,81 @@ title: "Customize documentation templates"
 date: last-modified
 ---
 
-Edit templates that get used for model documentation or for validation reports to configure templates for specific use cases or where the existing templates supplied by ValidMind need to be customized. 
+The {{< var vm_risk >}} offers robust and fully customizable templates for model documentation and validation reports. Easily configure these templates via multiple convenient methods to suit your custom needs. 
 
-## About documentation templates
-
-Documentation templates are stored as YAML files that you can edit directly in the online editor using the advanced **Edit Raw** button. We also provide an simplified editing experience with the **Edit** button. These templates are versioned and saving a documentation template after making changes or reverting to a previous version state always creates a new version. 
+  - Templates are versioned and saving a documentation template after making changes or reverting to a previous version state always creates a new version, or you can duplicate templates to create new independent templates in your library. 
+  - Documentation templates are stored as YAML files edited directly using the advanced editor, allowing you full control over the minitua such as desired guidelines. 
+  - We also provide a simplified editing experience to allow you to easily construct and rearrange the outline of your template.
 
 ## Prerequisites
 
-The template you want to edit must have been added to the {{< var vm_platform >}} already.
+- To configure templates, you must hold the `Customer Admin` role^1^. 
+- The template you want to edit must exist in the {{< var vm_platform >}}^2^.
+
+::: {.column-margin}
+**Learn more:**
+
+^1^ [Users (Supported roles)](onboarding-users.qmd#supported-roles)
+
+^1^ [Manage users (User roles)](manage-users.qmd#user-roles)
```

Comment:
```suggestion
^1^ [Users (Supported roles)](onboarding-users.qmd#supported-roles) and [Manage users (User roles)](manage-users.qmd#user-roles)
```

Duplicate footnote numbering looks off to me, I'd avoid by putting the related topics together. (If the left indent is an issue, you could always add a `<br>&ensp;` — no less hackey than our manual footnote numbering. 🙂 )

Reply:
I'll make a note to adjust this in my restructuring PR for duplicate entries (already playing around with some options) but I'm going to leave this in here for now.